src/types/project.types.ts

```diff
@@ -130,10 +145,15 @@ export interface NfrProject {
   activeTo: string;
 }
 
+export const emptyAffiliation: ProjectOrganization = {
+  type: 'Organization',
+  id: '',
+  labels: { no: '' },
+};
+
 export const emptyProjectContributor: ProjectContributor = {
-  type: 'ProjectParticipant',
   identity: { type: 'Person', id: '', firstName: '', lastName: '' },
-  affiliation: { type: 'Organization', id: '', labels: {} },
+  roles: [{ type: 'ProjectParticipant', affiliation: emptyAffiliation }],
```

Comment:
Er det ikke natrulig at denne også er tom?
Eller kompliserer det noe annet?
```suggestion
  roles: [],
```